web/html/src/manager/virtualization/HypervisorCheck.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as React from 'react';
+
+import { Messages } from 'components/messages';
+import { Utils as MessagesUtils } from 'components/messages';
+
+type Props = {
+    hypervisor: string,
+    foreignEntitled: boolean,
+};
+
+export function HypervisorCheck(props: Props) {
+    if (props.foreignEntitled) {
+        return null;
+    }
+
+    const virtMissing = props.hypervisor.includes("'virt' __virtual__ returned False");
+    if (virtMissing) {
+        const virtWarning = MessagesUtils.error(t("Please install libvirt python module."));
```

Comment:
Renaming `virtWarning` to `virtError` sounds better as long as it used for `MessagesUtils.error`.